sdk/quantum/quantum-jobs/samples/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "azure-quantum-jobs-samples-js",
```

Comment:
Regarding samples... restructuring required in a future PR.

The current structure we have for samples folder - https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/template/template/samples

JS and TS samples are to be separated in folders as shown in the above link.

Based on the metadata in the `readme` of those folders, the samples get published at `docs.microsoft.com`
Example: https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/service-bus-typescript/